Cargo.toml

```diff
@@ -16,7 +16,9 @@ nix = "0.17"
 dlib = "0.4"
 lazy_static = "1.0"
 memmap = "0.7"
-andrew = { version = "0.3.0", optional = true }
+raqote = "0.8"
```

Comment:
It would nice if the drawing library remained optional. Some users of SCTK do not need decorations drawn by the library.